bounce/server/api/membership.py

```diff
@@ -29,16 +29,18 @@ class MembershipEndpoint(Endpoint):
         # Decode the club name
         club_name = unquote(club_name)
         user_id = request.args.get('user_id', None)[0]
-        editor_role = request.args.get('access')
```

Comment:
Bruno said not to get the editor role from args and we should get it from body, I'll make this change.